src/main/service/AdminService.java

```diff
@@ -0,0 +1,36 @@
+package main.service;
+
+import main.entity.Ticket;
+import main.entity.User;
+import main.service.abstracts.AbstractUser;
+
+public class AdminService extends AbstractUser {
+
+    public void checkTicket(User user) {
```

Comment:
The same comments like for ClientService

Reply:
The AdminService class handles admin-specific logic, like managing tickets. Its name makes it clear what the class is for and keeps the code organized and focused on its purpose.

Why  AbstractUser?
AbstractUser acts as a blueprint for user-related services, ensuring all services (e.g., AdminService, ClientService) follow the same structure. This makes it easier to add new services in the future without changing existing code.